docker-compose.yml

```diff
@@ -0,0 +1,85 @@
+version: "3.8"
+services:
+  backend :
+    container_name: backend
+    build:
+      context: .
+      dockerfile: Dockerfile_Backend
+      args:
+        APP_ENV: development
+    ports:
+      - "8000:8000"
+    networks:
+      - application-network
+    depends_on:
+      - database
+    environment:
+      - POSTGRES_PASSWORD=dev_password
+      - POSTGRES_USER=postgres
+      - POSTGRES_SERVER=database
+  frontend:
+    container_name: frontend
+    build:
+      context: .
+      dockerfile: Dockerfile_Frontend
+    ports:
+      - "3000:3000"
+    networks:
+      - application-network
+    depends_on:
+      - backend
+      - database
+    environment:
+      - NEXT_PUBLIC_BACKEND_URL=http://backend:8000
```

Comment:
should this be 0.0.0.0:8000?

Reply:
no it was intentional to ensure that we are not exposing it to everyone using port 8000. did you try and rebuild all of your images? Are you available for a quick teams chat?